README.adoc

```diff
@@ -220,6 +220,7 @@ without operational burden or harming the cluster's performance
 * Solution for the https://en.wikipedia.org/wiki/Head-of-line_blocking["head of line"] blocking problem where continued failure of a single message, prevents progress for messages behind it in the queue
 * Per `key` concurrent processing, per partition and unordered message processing
 * Offsets committed correctly, in order, of only processed messages, regardless of concurrency level or retries
+* Batch version fo the API to process batches of messages in parallel instead of single messages.
```

Comment:
investigate build speed change

Reply:
might be due to the slower commit rate - 5s vs 100ms - a lot of sets "wait for one cycle", which when there's no work might be the full 5 seconds. Plus on CI the tests are sequential.

Reply:
deferring to
- #226